rand_pcg/tests/lcg64xsh32.rs

```diff
@@ -1,6 +1,18 @@
 use rand_core::{RngCore, SeedableRng};
 use rand_pcg::{Lcg64Xsh32, Pcg32};
 
+#[test]
+fn test_lcg64xsh32_advancing() {
+    let seed = Default::default();
+    let mut rng1 = Lcg64Xsh32::from_seed(seed);
+    let mut rng2 = Lcg64Xsh32::from_seed(seed);
+    for _ in 0..20 {
+        rng1.next_u64();
+    }
+    rng2.advance(20);
```

Comment:
This is a 32-bit generator, so `next_u64` actually steps twice internally. This is the source of your confusion?

Reply:
That's it! Accordingly, I think 624d24e8ae0043bfbd59c369e1741ab3bc848a64 should be reverted, and the test should use `next_u32`.